README.md

```diff
@@ -102,10 +104,169 @@ Refer to the relevant CI pipeline scripts in `/.github/workflows`.
+
+#### Skills
+
+A similar view to the Staff's version of skills is displayed, with the addition of 3 buttons:
+- `Create New Skill`
+- `Edit`
+- `Deactivate`
+
+<img src="assets/hr/skills.png" alt="HR Skills" />
+
+##### Create New Skill
+
+Selecting the `Create New Skill` option brings up a page to furnish the skill name and description for a new skill. By default, new skills are set to active.
+
+<img src="assets/hr/create_skill.png" alt="HR Create Skill" />
+
+#### Edit
+
+Selecting the `Edit` option on a skill brings up its respective edit skill page. Changes in the fields will then be saved after using the `Update Skill` button.
+
+<img src="assets/hr/edit_skill.png" alt="HR Edit Skill" />
+
+#### Deactivate
+
+Selecting the `Deactivate` option on a skill brings up a confirmation window to "soft-delete" a skill (i.e. set the skill to inactive). This would prevent Staff users from viewing the respective skill.
+
+<img src="assets/hr/deactivate_skill.png" alt="HR Deactivate Skill" />
 
 <p align="right">(<a href="#top">back to top</a>)</p>
 
+### Courses
+
+A similar view to the Staff's version of skills is displayed, with the addition of 1 button:
+- `Edit`
+
+Also, course statuses are displayed via color coded tags (`Active`, `Pending` and `Retired`) instead of the course registration status.
+
+<img src="assets/hr/courses.png" alt="HR Courses" />
+
+#### Edit
+
+Selecting the `Edit` option on a course brings up its respective edit course page. Skills can be assigned/removed using the dropdown box and crosses.
+
+<img src="assets/hr/edit_course.png" alt="HR Edit Course" />
+
+### Job Roles
+
+A similar view to the Staff's version of job roles is displayed, with the addition of 3 buttons:
+- `Create New Job`
+- `Edit`
+- `Deactivate`
+
+<img src="assets/hr/jobs.png" alt="HR Jobs" />
+
+##### Create New Job
+
+Selecting the `Create New Job` option brings up a page to furnish the job name and description for a new job. In addition, skills can be assigned in this page. By default, new jobs are set to active.
+
+<img src="assets/hr/create_job.png" alt="HR Create Job" />
+
+#### Edit
+
+Selecting the `Edit` option on a job role brings up its respective edit job role page. Changes in the fields will then be saved after using the `Update Job` button.
+
+<img src="assets/hr/edit_job.png" alt="HR Edit Job" />
+
+#### Deactivate
+
+Selecting the `Deactivate` option on a job role brings up a confirmation window to "soft-delete" a job role (i.e. set the job role to inactive). This would prevent Staff users from viewing the respective job role.
+
+<img src="assets/hr/deactivate_job.png" alt="HR Deactivate Job" />
+
+### Manager Features
+
+Currently, none of the first release features are unique to Managers, and therefore no notable features are available under this role. Future features such as viewing the skills of their team members will be added in subsequent releases.
```

Comment:
Currently, none of the first release features **is** unique to Managers, and therefore no notable features **is** available under this role. Future features such as viewing the skills of their team members will be added in subsequent releases.